CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## [release-49] - 2019-02-26
+
+- Admin user can deactivate and activate suppliers on frameworks
+- Added code to automate generation of users with late tasks list
```

Comment:
Would "Added code to automate generation of the list of users with late tasks" be more readable?

Reply:
👍